src/Readers/AbstractReader.js

```diff
@@ -40,12 +40,13 @@ class AbstractReader {
    * @return {{__FILE__, __PATH__}}
    */
   parsePath(source) {
-
     const __FILE__ = path.basename(source);
     let __PATH__;
 
     let parsedURL = url.parse(source);
-    if (parsedURL.protocol) {
+
+    // url parse can find protocol in Windows-style path, so we check it
+    if (parsedURL.protocol && !parsedURL.protocol.match(/^(\w\:)?$/i)) {
```

Comment:
I understand what the function do. My point is that it does the right thing even for windows paths (given that it has slashes reverted in `_includeSource` before that).

So it returns correct pair of path/file for `c:/path/to/my/file` for example. And thus the change that you made a) is not necessary and b) is harmful in case there are single char named protocols (why not?).